pip_review/__main__.py

```diff
@@ -1,18 +1,15 @@
 from __future__ import absolute_import
-import os
 import re
 import argparse
 from functools import partial
 import logging
 import json
 import sys
 import pip
-import subprocess
```

Comment:
`subprocess` is actually used. But you were probably testing on Python 3, which allows you to remove it here because `subprocess` is also imported in the `PY3` branch on line 16. I removed the latter instead.